testsuite/tests/lib-dynlink-csharp/Makefile

```diff
@@ -30,7 +30,9 @@ all: prepare bytecode bytecode-dll native native-dll
 prepare:
 	@if $(SUPPORTS_SHARED_LIBRARIES); then \
```

Comment:
I don't like `which` because it's an external command (it even used to be a shell script on older Unix systems) that has a tendency to get subtly out of sync with the way the shell looks for commands.

The built-in shell command that provides the sames functionality (with a different interface) is called `type`.